backend/scripts/create-idol-images-pr.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-console */
+import { Octokit } from '@octokit/rest';
+import { spawnSync } from 'child_process';
+
+function runCommand(program: string, ...programArguments: readonly string[]) {
+  const programArgumentsQuoted = programArguments
+    .map((it) => (it.includes(' ') ? `"${it}"` : it))
+    .join(' ');
+  console.log(`> ${program} ${programArgumentsQuoted}`);
+  return spawnSync(program, programArguments, { stdio: 'inherit' });
+}
+
+const main = async () => {
+  console.log('Creating PR...');
+
+  // Create commit
+  runCommand('git', 'config', '--global', 'user.name', 'dti-github-bot');
+  runCommand('git', 'config', '--global', 'user.email', 'admin@cornelldti.org');
+  const gitBranch = 'dti-github-bot/pull-from-idol-images';
+  const commitMessage = '[bot] Automatically pull images from IDOL';
+  runCommand('git', 'add', '../new-dti-website/public/team');
+  runCommand('git', 'fetch', '--all');
+  runCommand('git', 'checkout', 'main');
+  runCommand('git', 'checkout', '-b', gitBranch);
+  if (runCommand('git', 'commit', '-m', commitMessage).status === 0) {
+    if (runCommand('git', 'push', '-f', 'origin', gitBranch).status !== 0) {
+      runCommand('git', 'push', '-f', '--set-upstream', 'origin', gitBranch);
+    }
+  }
+
+  // Create PR
+  const octokit = new Octokit({
+    auth: `token ${process.env.BOT_TOKEN}`,
+    userAgent: 'cornell-dti/big-diff-warning'
+  });
+  const prBody = `
+  ## Summary
+
+  This is a PR auto-generated from running \`yarn workspace backend run pull-from-idol-images\` and
+  \`yarn workspace backend run create-idol-images-pr\`.
+
+  It updates the new-dti-website with latest images from IDOL.
+
+  Please verify that the images look fine on the website and that the image sizes aren't too large.`;
```

Comment:
this in and of itself makes this automation seem not too useful - there's about 0% chance a headshot that someone uploads is the right size (iPhone photo size is way larger than 20KB which is expected). in which case the reviewer has to pull the branch, run the image-cli, push the commit  

and expecting the user to compress the headshot first just seems like a bad UX imo

Reply:
We have a follow up item for image compression here!